tools/sil-opt/SILOpt.cpp

```diff
@@ -498,12 +504,12 @@ int main(int argc, char **argv) {
 
   bool HadError = CI.getASTContext().hadError();
 
+  PrintDiags.setSuppressOutput(false);
+
   // If we're in -verify mode, we've buffered up all of the generated
   // diagnostics.  Check now to ensure that they meet our expectations.
```

Comment:
Is this comment still accurate with the refactoring you've done? I don't think the final processing actually happens here.